server/integration_tests/standalone/adversarial.py

```diff
@@ -323,10 +333,10 @@ def run_query(self, query: str) -> Result:
     try:
       resp = requests.post(
           self.base_url +
-          f'/api/explore/detect-and-fulfill?q={query}&llm_api={self.llm_api}',
+          f'/api/explore/detect-and-fulfill?q={query}&llm_api={self.llm_api}&dc={sdg_param}',
```

Comment:
Do we need to include the `dc` param here since it's already encoded in the json payload?

Reply:
Updated